utils/s2n_str.c

```diff
@@ -17,10 +17,18 @@
 #include <sys/param.h>
 
 char *s2n_strcpy(char *buf, char *last, const char *str) {
+
+/* CBMC pointer checks need to be disabled to compare buf and last for
+ * the case where they are the same. */
```

Comment:
Its not just this.  Its actually a precondition that buf and last be pointers into the same array; otherwise this is actually undefined behaviour in C.

Reply:
Good catch! I implemented your previous comment instead.